virt_lightning/shell.py

```diff
@@ -169,29 +145,25 @@ def iconify(v):
         else:
             return symbols.CROSS.value
 
-    while True:
-        for status in get_status(hv, context):
-            results[status["name"]] = {
-                "name": status["name"],
-                "ipv4": status["ipv4"] or "waiting",
-                "context": status["context"],
-                "username": status["username"],
-                "ssh_ping": iconify(status["ssh_ping"]),
-            }
+    for status in get_status(hv, context):
+        results[status["name"]] = {
+            "name": status["name"],
+            "ipv4": status["ipv4"] or "waiting",
```

Comment:
You can change that code to `status.get("ipv4", "waiting")`

Reply:
With the current code, `status["ipv4"]` can be `None`, `get("ipv4", "foo")` would return `None`, not `foo`.